packages/fiori/src/MediaGallery.ts

```diff
@@ -348,26 +343,26 @@ class MediaGallery extends UI5Element {
 		return this._getOverflowSize(columnHeight, columnsCount);
 	}
 
-	_toggleDisplaySquareSize(enable) {
-		this._display.style.width = ""; // restore default width
+	_toggleDisplaySquareSize(enable: boolean) {
+		this._display!.style.width = ""; // restore default width
 
 		if (enable) {
 			const marginSize = MediaGallery.THUMBNAIL_MARGIN,
-				width = this._display.offsetWidth;
+				width = this._display!.offsetWidth;
 
-			let availableHeight = this.getDomRef().offsetHeight - (2 * marginSize);
+			let availableHeight = this.getDomRef()!.offsetHeight - (2 * marginSize);
 			if (this.effectiveLayout === MediaGalleryLayout.Vertical) {
 				availableHeight -= (MediaGallery.THUMBNAIL_HEIGHT + marginSize);
 			}
 
 			if (width > availableHeight) {
 				// set to square
-				this._display.style.width = `${availableHeight}px`;
+				this._display!.style.width = `${availableHeight}px`;
 			}
 		}
 	}
 
-	_toggleMainItem9x16size(enable) {
+	_toggleMainItem9x16size(enable: boolean | null) {
```

Comment:
leave only boolean and fix because it should return boolean _shouldHaveWideDisplay